doc/whats_new/v0.23.rst

```diff
@@ -140,6 +142,17 @@ Changelog
    exclusively choose the components that explain the variance greater than
    `n_components`. :pr:`15669` by :user:`Krishna Chaitanya <krishnachaitanya9>`
 
+
+- |Enhancement| added a new approximate solver (randomized SVD, available with
+  `eigen_solver='randomized'`) to :class:`decomposition.KernelPCA`. This
+  drastically accelerates computation when the number of samples gets much
+  larger than the desired number of components.
+  For example with 10000 samples, a fairly accurate kPCA model using 100
+  principal components can be trained more than 10 times faster with
+  `eigen_solver='randomized'` than with the default `eigen_solver='full'`
+  solver.
```

Comment:
we usually avoid such details in the what'snew and write them in the UG

Reply:
ok